jerry-debugger/jerry-client-ws.py

```diff
@@ -728,6 +759,35 @@ def parse_source(debugger, data):
     else:
         logging.debug("No pending breakpoints")
 
+def printSource(self, line_num):
+    last_bp = self.debugger.last_breakpoint_hit
+    lines = last_bp.function.source.split('\n',last_bp.function.source.count('\n') - 1)
```

Comment:
Even if the source is an empty string, the array should have at least one value. if "".split("\n") return with an array which contains an empty string: `[ "" ]` that is ok. If it returns with an empty array `[]` than we should manually add a "" to this array.

And one more question: why do we need a second argument to split?